idea_town/experiments/views.py

```diff
@@ -14,7 +14,7 @@ class ExperimentViewSet(viewsets.ModelViewSet):
     Returns a list of all Experiments in the system.
     """
     queryset = Experiment.objects.all()
-    serializer_class = ExperimentSerializer
+    serializer_class = ExperimentDeepSerializer
```

Comment:
Okay, I see what's up: The experiments set gets loaded on startup, and this API change includes all the data so that the same set can be used on detail views. 

IMO the better way is to use the shallow serializer as-is, then request the `details_url` resource on demand when an experiment detail view is visited. I might try ginning up a PR to this branch that does that.

The full set will mostly contain data the user will rarely see (i.e. because they never navigate to all experiments). So, the API response will be mostly wasted work (i.e. one DB query for the list of experiments, then an additional DB query for each experiment to populate details, then the whole thing bundled up as a JSON response)